services/director-v2/src/simcore_service_director_v2/modules/db/repositories/services_environments.py

```diff
@@ -46,11 +46,12 @@ async def get_vendor_secrets(
     def is_vendor_secret_identifier(cls, identifier: str) -> bool:
         return identifier.startswith(VENDOR_SECRET_PREFIX)
 
-    async def get_user_role(self, user_id: UserID):
+    async def get_user_role(self, user_id: UserID) -> UserRole:
         async with self.db_engine.acquire() as conn:
-            return UsersRepo().get_role(conn, user_id=user_id)
+            user_role = await UsersRepo().get_role(conn, user_id=user_id)
+            return UserRole(user_role)
```

Comment:
sorry, but this makes no sense. You put ANy in the get_role function and then you cast it? what is the idea here